packages/request-client.js/package.json

```diff
@@ -49,6 +49,7 @@
     "@requestnetwork/epk-signature": "0.5.6",
     "@requestnetwork/multi-format": "0.2.3",
     "@requestnetwork/request-logic": "0.8.2",
+    "@requestnetwork/smart-contracts": "0.2.0",
```

Comment:
have you checked the size of the final bundle? Just curious if it increases, since we add new dependencies...

Reply:
No, but we share you concern with @rittme.
I Jira this, so we can handle it later.